repository/src/main/java/org/alfresco/repo/node/archive/GetArchivedNodesCannedQuery.java

```diff
@@ -119,4 +120,26 @@ private void preload(List<NodeRef> nodeRefs)
                         + (System.currentTimeMillis() - start) + " msecs");
         }
     }
+
+    @Override
+    protected Pair<Integer, Integer> getTotalResultCount(List<ArchivedNodeEntity> results)
+    {
+        Object paramBeanObj = super.getParameters().getParameterBean();
+        if (paramBeanObj == null)
+            throw new NullPointerException("Null GetArchivedNodes query params");
```

Comment:
I think the log is generic or misleading

Reply:
Since the query parameters are required...Maybe it could read as "Required parameters not provided for GetArchivedNodes canned query".